### DISCUSSION TITLE
Custom color only for backlinks (linked and unlinked)

### DISCUSSION THREAD

**Discussion by TonkatsuRa** [AnubisNekhet/AnuPpuccin#146] [Ideas]
I'd love to change the colours of the backlinks (linked and unlinked) in the AnuPpuccin Theme.
So far I need to change the accent color to change that, while changes the color of the frame and just turns the backlinks in two shades of that color.

Please add, that I can for example make a linked backlink "green" and an unlinked one "red", while keeping a black outer frame.

**Comment by RedLilith** [comment]
This is what I use to add a custom color to my backlinks. Hope it helps

```
/* << EDITING link color INTERNAL >>  */
.cm-s-obsidian span.cm-hmd-internal-link {
  font-weight: 400;
  color: rgb(197, 146, 222)  !important;
}

/* << READING link color INTERNAL >>  */
.markdown-preview-view .internal-link {
  font-weight: 400;
  color: rgb(197, 146, 222)  !important;
}
```
